NineChronicles.Headless/BlockChainService.cs

```diff
@@ -211,30 +215,42 @@ public UnaryResult<byte[]> GetStateBySrh(byte[] addressBytes, byte[] stateRootHa
             IEnumerable<byte[]> addressBytesList,
             byte[] stateRootHashBytes)
         {
-            var stateRootHash = new HashDigest<SHA256>(stateRootHashBytes);
+            var started = DateTime.UtcNow;
+            var sw = new Stopwatch();
+            sw.Start();
             var result = new Dictionary<byte[], byte[]>();
             List<Address> addresses = new List<Address>();
             foreach (var b in addressBytesList)
             {
                 var address = new Address(b);
                 if (_memoryCache.TryGetValue(address.ToString(), out byte[] cached))
                 {
-                    result.TryAdd(b, cached);
+                    result.TryAdd(b, MessagePackSerializer.Serialize(cached, _lz4Options));
                 }
                 else
                 {
                     addresses.Add(address);
                 }
             }
-            IReadOnlyList<IValue> values = _blockChain.GetAccountState(_blockChain.Tip.Hash).GetStates(addresses);
-            for (int i = 0; i < addresses.Count; i++)
+            sw.Stop();
+            Log.Information("[GetSheets]Get sheet from cache count: {Count}, not Cached: {Count2}, Elapsed: {Elapsed}", result.Count, addresses.Count, sw.Elapsed);
```

Comment:
When looking at these logs with a filter in Loki(Grafana), it will be hard to know what `Count` and `Count2` means.

```suggestion
            Log.Information("[GetSheets]Get sheet from cache count: {CachedCount}, not Cached: {CacheMissedCount}, Elapsed: {Elapsed}", result.Count, addresses.Count, sw.Elapsed);
```